daml-script/export/it/src/test/resources/interface/Export.daml

```diff
@@ -1,6 +1,3 @@
--- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
--- SPDX-License-Identifier: Apache-2.0
-
```

Comment:
Adding in the directory a file `NO_AUTO_COPYRIGHT` exempts us from putting a copyright header.
Easier when comparing the output.